### ISSUE TITLE
go-task new: Automatically add the MAINTAINERS.md file from data in packager file

### ISSUE THREAD

**Issue by Justinzobel** [getsolus/packages#1233]
Automatically add the MAINTAINERS.md file from data in packager file 
This would make adding a new packager more automated and easier.

David Notes:
- Would probably need two operations
  - `maintain-add` to add yourself to the existing MAINTAINERS file, below any existing entry
  - `maintain-new` to replace the existing file with one containing only your own info

**Comment by androidnisse** [comment]
Yes, it would be great to have an easier way to create that file.
